[2022-12-04 09:56:05,774] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.save_to_parquet scheduled__2022-10-30T22:20:00+00:00 [queued]>
[2022-12-04 09:56:05,828] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.save_to_parquet scheduled__2022-10-30T22:20:00+00:00 [queued]>
[2022-12-04 09:56:05,842] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 09:56:05,858] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-12-04 09:56:05,859] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 09:56:05,926] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): save_to_parquet> on 2022-10-30 22:20:00+00:00
[2022-12-04 09:56:05,935] {standard_task_runner.py:52} INFO - Started process 2918 to run task
[2022-12-04 09:56:05,974] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'save_to_parquet', 'scheduled__2022-10-30T22:20:00+00:00', '--job-id', '2741', '--raw', '--subdir', 'DAGS_FOLDER/AirflowClean2.py', '--cfg-path', '/tmp/tmpupbw9jnm', '--error-file', '/tmp/tmprnwqzjf1']
[2022-12-04 09:56:05,977] {standard_task_runner.py:77} INFO - Job 2741: Subtask save_to_parquet
[2022-12-04 09:56:06,226] {logging_mixin.py:109} INFO - Running <TaskInstance: dag1.save_to_parquet scheduled__2022-10-30T22:20:00+00:00 [running]> on host 9e44869ff3ef
[2022-12-04 09:56:06,440] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Fadlil
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=save_to_parquet
AIRFLOW_CTX_EXECUTION_DATE=2022-10-30T22:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-30T22:20:00+00:00
[2022-12-04 09:56:06,583] {python.py:175} INFO - Done. Returned value was: None
[2022-12-04 09:56:06,644] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=save_to_parquet, execution_date=20221030T222000, start_date=20221204T095605, end_date=20221204T095606
[2022-12-04 09:56:06,887] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-04 09:56:07,887] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
